gene2phenotype_project/gene2phenotype_app/views/disease.py

```diff
@@ -138,6 +138,36 @@ def list(self, request, *args, **kwargs):
 
         return Response(response_data)
 
+class ExternalDisease(BaseView):
+    """
+        Returns the disease for an external disease ID.
+        External sources can be OMIM or Mondo.
+
+        Args:
+            (str) external disease id: the disease ID of the external source (OMIM/Mondo)
+
+        Returns:
+            Response object is a dict with the following structure
+                    (string) disease: the disease name as represented in the source
+                    (string) identifier: disease ID
+                    (string) source: source name
+    """
+    serializer_class = GeneDiseaseSerializer
+
+    def get_queryset(self):
+        disease_id = self.kwargs['ext_id']
+
```

Comment:
Also not sure if this is important but because this might be public maybe in the future. 
Can we make sure the ext_id either startswith MONDO or just digits for now to keep it constrained to MONDO and OMIM.

Reply:
Changed to only query the db if ext_id matches Mondo or OMIM format.
If disease not found in G2P or format is different, returns `"message": "No matching disease found for: GNC0971015"`